geniza/corpus/tests/test_corpus_views.py

```diff
@@ -203,6 +203,24 @@ def test_pgp_metadata_for_old_site():
 
 
 class TestDocumentSearchView:
+    def test_document_suppression(self, document, suppressed_document):
+        SolrClient().update.index(
+            [
+                document.index_data(),  # no scholarship records
+                suppressed_document.index_data(),  # one scholarship record
+            ],
+            commit=True,
+        )
+        docsearch_view = DocumentSearchView()
+        docsearch_view.request = Mock()
+
+        # keyword search param
+        docsearch_view.request.GET = {"q": ""}
+        qs = docsearch_view.get_queryset()
+        assert qs.count() == 1
+        assert document.id in [obj["pgpid"] for obj in qs]
+        assert suppressed_document.id not in [obj["pgpid"] for obj in qs]
```

Comment:
When I'm using a list like this more that once, I find it more readable to generate the list first. Something like this:

```suggestion
        resulting_pgpids = [obj["pgpid"] for obj in qs]
        assert document.id in resulting_pgpids
        assert suppressed_document.id not in resulting_pgpids
```